test/anyons.jl

```diff
@@ -18,3 +18,35 @@ using Test
     @test norm(curlA - 2π*DFTK.ρref_real_2D(x, y, M, σ)) < 1e-4
     @test abs(divA) < 1e-6
 end
+
+@testset "Anyons: check E11" begin
```

Comment:
What is E11 ?

Reply:
Some quantity defined in the paper, doesn't really matter what it is, it's just a test we reproduce the results in the paper